### ISSUE TITLE
Problem writing to/reading from a measurement after it has been dropped (0.9.0)

### ISSUE THREAD

**Issue by jshamash** [influxdb/influxdb#3035]
Running a fresh install of 0.9.0 stable on my mac (Yosemite).

```
# create the database
curl -G http://localhost:8086/query --data-urlencode "q=CREATE DATABASE my_db"

# write to my_measurement
curl -XPOST "http://localhost:8086/write?db=my_db&precision=ms" -d "my_measurement value=1 1418256001000"

# read from my_measurement. This gives the expected value.
curl -G http://localhost:8086/query --data-urlencode "db=my_db" --data-urlencode "q=select count(value) from my_measurement"
# {"results":[{"series":[{"name":"my_measurement","columns":["time","count"],"values":[["1970-01-01T00:00:00Z",1]]}]}]}

# drop my_measurement
curl -G http://localhost:8086/query --data-urlencode "db=my_db" --data-urlencode "q=drop measurement my_measurement"

# write to my_measurement
curl -XPOST "http://localhost:8086/write?db=my_db&precision=ms" -d "my_measurement value=1 1418256001000"

# read from my_measurement, this time an error occurs.
curl -G http://localhost:8086/query --data-urlencode "db=my_db" --data-urlencode "q=select count(value) from my_measurement"
# {"results":[{"error":"unknown field or tag name in select clause: value"}]}
```

Before the drop, everything behaves as expected. After the drop, when writing and then reading to the dropped measurement, I get the error "unknown field or tag name in select clause: value". I also noticed:

- After dropping `my_measurement` and then writing to `my_measurement`, if I do a `show measurements` I can see that `my_measurement` has in fact been created.
- After the drop, other measurements can be written to/read from with no issues.
- After the drop, if I delete the database and recreate it, I can write to/read from `my_measurement` again.

**Comment by indrekj** [comment]
I think this is already fixed (issue #2955), not released yet though

**Comment by otoolep** [comment]
Yes, fixed by #3002, which will be in 0.9.1.

In the meantime you can either build from source, or restart your system after dropping the measurement. A restart completes the drop.
